sdks/python/apache_beam/examples/complete/autocomplete_test.py

```diff
@@ -41,7 +37,7 @@ def assert_that(pcoll, matcher):
   def check_matcher(_, side_value):
     assert matcher(side_value)
     return []
-  singleton | beam.FlatMap(check_matcher, AsIter(pcoll))
+  singleton | beam.FlatMap(check_matcher, AsIter(pcoll))  # pylint: disable=expression-not-assigned
```

Comment:
I feel like `expression-not-assigned` is a good candidate for global disabling, given the use of `|` as a side-effecting expression in the SDK.